src/style.css

```diff
@@ -0,0 +1,123 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300px;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.body {
+  margin-top: 0;
+}
+
+.container {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-field.big {
+  width: 100%;
+  height: 70px;
+  background-color: #fff ;
+  border-radius: 4px;
+  border: solid 1px #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300px;
+  padding-left: 62px;
+}
+
+.search-field.small {
+  width: 100%;
+  height: 42px;
+  background-color: #fff ;
+  border-radius: 4px;
+  border: solid 1px #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
+  font-weight: 300px;
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+.search-field.big::placeholder {
```

Comment:
do not increase css specificity
use just one class instead of `search-field.big`

Reply:
fix all such cases